### DISCUSSION TITLE
osu!lazer always crashes on startup unless I open it in the background

### DISCUSSION THREAD

**Discussion by chiqki** [ppy/osu#22620] [Q&A]
### Type

Crash to desktop

### Bug description

So I've been trying to open osu!lazer for almost 6 months now and 99% of the time I couldn't click on anything. I could hear stuff, but the whole screen froze and I would have to AltF4 to get out. Just recently though, I figured out that if I clicked to open the game and then clicked on any other tab, the game would work if I waited a little while. I don't want to have to do that every single time though, and I wanted to know if there was a fix to this.

### Version

2023.207.0-lazer

### Logs

[performance.log](https://github.com/ppy/osu/files/10714636/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10714637/runtime.log)
[updater.log](https://github.com/ppy/osu/files/10714638/updater.log)
[database.log](https://github.com/ppy/osu/files/10714639/database.log)
[input.log](https://github.com/ppy/osu/files/10714640/input.log)
[network.log](https://github.com/ppy/osu/files/10714641/network.log)

**Comment by bdach** [comment]
Please attach Windows Event Viewer info by performing the following steps:
1. Type `Event viewer` in the Windows search bar (shortcut: <kbd>Windows</kbd>+<kbd>S</kbd>) and press <kbd>Enter</kbd>, or open the *Run...* button (<kbd>Windows</kbd>+<kbd>R</kbd>), type in `eventvwr` and press <kbd>Enter</kbd>.
2. The *Event viewer* window will appear. Expand *Windows Logs* on the left and choose *Application*. It might take a while to load.
3. In the *Actions* pane on the right click *Save All Events As...*, then choose a location to save to.
4. On the *Display Information* popup, select the second option, then the checkbox to show all available languages, then *English (UK)* (US would also do fine).
5. Append the resulting `.evtx` file to this discussion.

**Reply by bdach** [reply]
I can see some generic "access violation" exceptions in the files attached. Unfortunately those are usually scarce on details. The most I can recommend for now probably is trying to upgrade graphics device drivers.
